src/components/molecules/OTPInput/OTPInput.component.tsx

```diff
@@ -4,36 +4,70 @@ import OtpInput, { OtpInputProps } from 'react-otp-input'
 import { useTheme } from '../../../styles/theme'
 import { Typography } from '../../atoms/Typography/Typography.component'
 
-export interface OTPInputProps extends OtpInputProps {
+export enum OTPInputType {
+  Box = 'box',
+  Underline = 'underline'
+}
+export interface OTPInputProps extends Omit<OtpInputProps, 'onChange' | 'numInputs'> {
   errorMessage?: string
+  style?: CSSStyleDeclaration
+  cellTextStyle?: CSSStyleDeclaration
```

Comment:
@duongjitera. Does it support className ? Because currently generator will generate the style as class

Reply:
I'll fix it